wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/MockHardwareExtension.java

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
Is this a related change?

Reply:
Yes. This file is needed to run the command tests, but we still need a copy in the old test suite.